src/fe-common/irc/fe-events.c

```diff
@@ -252,13 +252,18 @@ static void event_pong(IRC_SERVER_REC *server, const char *data, const char *nic
 static void event_invite(IRC_SERVER_REC *server, const char *data,
 			 const char *nick, const char *addr)
 {
-	char *params, *channel;
+	char *params, *invited, *channel;
 
 	g_return_if_fail(data != NULL);
 
-	params = event_get_params(data, 2, NULL, &channel);
-	signal_emit("message invite", 4,
-		    server, get_visible_target(server, channel), nick, addr);
+	params = event_get_params(data, 2, &invited, &channel);
+	if (server->nick_comp_func(invited, server->nick) == 0) {
```

Comment:
Out-of-scope but this function seems very underused for own-nick comparisons in other parts of the codebase, those seem to prefer `g_ascii_strcasecmp` which is going to be accurate than this one.